README.md

```diff
@@ -1 +1,32 @@
 # azuretpr
+
+Specification for Kubernetes clusters running in Giantnetes on Azure and managed by [azure-operator](https://github.com/giantswarm/azure-operator).
+
+## Getting Project
+
+Clone the git repository: https://github.com/giantswarm/azuretpr.git
+
+### How to build
+
+Build it using the standard `go build` command.
```

Comment:
Both commands build the same packages but I've reverted because I think it's simpler.

```
go build -v github.com/giantswarm/azuretpr                                                                                        
github.com/giantswarm/azuretpr/spec
github.com/giantswarm/azuretpr
```

```
go build -v $(glide novendor)
github.com/giantswarm/azuretpr/spec
github.com/giantswarm/azuretpr
```